Graphics/Implicit/Definitions.hs

```diff
@@ -229,104 +236,101 @@ data SharedObj obj vec
 
 instance (Show obj, Show vec) => Show (SharedObj obj vec) where
   showsPrec = flip $ \case
-     Empty                   -> showCon "emptySpace"
```

Comment:
I'm pretty sure eating this spacing makes things worse.

Reply:
Yes, that's one of the few aspects of ormolu I'm not terribly fond of.  I do just realize though that it has bothered me a lot more in the beginning, so you *can* get used to it.

Reply:
The reason why alignment is pooped on by ormolu is that there is no way of deciding automatically what the developer wants to align at.  But we could lobby for eventually getting this: https://github.com/tweag/ormolu/issues/612